runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -154,8 +183,17 @@ private void insertClanRankIcon(final SetMessage message)
 		if (rank != null && rank != ClanMemberRank.UNRANKED)
 		{
 			int iconNumber = clanManager.getIconNumber(rank);
-			message.getMessageNode()
-				.setSender(message.getMessageNode().getSender() + " <img=" + iconNumber + ">");
+			String img = "<img=" + iconNumber + ">";
+			if (message.getType().equals(ChatMessageType.CLANCHAT))
```

Comment:
Enum values can be compared directly with `==`.
```suggestion
			if (message.getType() == ChatMessageType.CLANCHAT)
```